includes/admin/views/omise-page-settings.php

```diff
@@ -136,7 +136,9 @@
 
 		<h3><?php _e( 'Payment Methods', 'omise' ); ?></h3>
 		<?php if ($settings['account_country']) : ?>
-			<p><?php _e( 'The table below is a list of available payment methods that you can enable in your WooCommerce store.', 'omise' ); ?></p>
+			<!--  // if account_country is exist that mean key is valid -->
```

Comment:
```suggestion
			<!--  // account_country exists when API key is valid -->
```
maybe something like this? and also the same comment in another file

Reply:
nice catch here updated 🙏 -> https://github.com/omise/omise-woocommerce/pull/290/commits/940e696c71b8bcea4a4e8624feb1c493667f5c20